doc/python/axes.md

```diff
@@ -215,76 +236,144 @@ fig.update_yaxes(ticks="outside", tickwidth=2, tickcolor='crimson', ticklen=10,
+
+Here is an example of rotating the x-axis tick labels by 45 degrees, and customizing their font properties, in a faceted histogram figure created using Plotly Express.
+
+```python
+import plotly.express as px
+df = px.data.tips()
+
+fig = px.histogram(df, x="sex", y="tip", histfunc='sum', facet_col='smoker')
+fig.update_xaxes(tickangle=45, tickfont=dict(family='Rockwell', color='crimson', size=14))
+
+fig.show()
+```
+
+#### Enumerated Ticks with Tickvals and Ticktext
+
+The `tickvals` and `ticktext` axis properties can be used together to display custom tick label text at custom locations along an axis. They should be set to lists of the same length where the `tickvals` list contains positions along the axis, and `ticktext` contains the strings that should be displayed at the corresponding positions.
+
+Here is an example.
 
 ```python
 import plotly.graph_objects as go
+import pandas as pd
 
+# Load and filter Apple stock data for 2016
+apple_df = pd.read_csv(
+    "https://raw.githubusercontent.com/plotly/datasets/master/finance-charts-apple.csv",
+    parse_dates=["Date"],
+    index_col="Date"
+)
+
+apple_df_2016 = apple_df["2016"]
+
+# Create figure and add line
 fig = go.Figure()
-fig.add_trace(go.Scatter(y=[1, 0]))
+fig.add_trace(go.Scatter(
+    x=apple_df_2016.index,
+    y=apple_df_2016["AAPL.High"],
+    mode="lines"
+))
 
-fig.update_xaxes(title_text='Time')
-fig.update_yaxes(title_text='Value A')
+# Set custom x-axis labels
+fig.update_xaxes(
+    ticktext=["End of Q1", "End of Q2", "End of Q3", "End of Q4"],
+    tickvals=["2016-04-01", "2016-07-01", "2016-10-01", apple_df_2016.index.max()],
+)
+
+# Prefix y-axis tick labels with dollar sign
+fig.update_yaxes(tickprefix="$")
+
+# Set figure title
+fig.update_layout(title_text="Apple Stock Price")
 
 fig.show()
 ```
 
-### Set axis title position
 
-This example sets `standoff` attribute to cartesian axes to determine the distance between the tick labels and the axis title. Note that the axis title position is always constrained within the margins, so the actual standoff distance is always less than the set or default value. By default [automargin](https://plot.ly/python/setting-graph-size/#automatically-adjust-margins) is `True` in Plotly template for the cartesian axis, so the margins will be pushed to fit the axis title at given standoff distance.
+### Axis lines: grid and zerolines
+
+##### Toggling Axis grid lines
+
+Axis grid lines can be disabled by setting the `showgrid` property to `False` for the x and/or y axis.
+
+Here is an example of setting `showgrid` to `False` in the graph object figure constructor.
 
 ```python
-import plotly.graph_objects as go
+import plotly.express as px
 
-fig = go.Figure(go.Scatter(
-    mode = "lines+markers",
-    y = [4, 1, 3],
-    x = ["December", "January", "February"]))
+fig = px.line(y=[1, 0])
+fig.update_layout(xaxis_showgrid=False, yaxis_showgrid=False)
```

Comment:
I would use the fuller `update_*axes()` here, otherwise this can't copy-paste into facetted figures